skrf/io/mdif.py

```diff
@@ -365,7 +365,10 @@ def write(ns : NetworkSet,
               filename : str,
               values: dict | None = None,
               data_types: dict | None = None,
-              comments = None):
+              *,
+              comments: str | None = None,
+              skrf_comment: bool = True,
+              ads_compatible: bool = True):
```

Comment:
I thought about:

```python
@staticmethod
    def write(ns : NetworkSet,
              filename : str,
              values: dict | None = None,
              data_types: dict | None = None,
              comments: str | None = None,
              *,
              skrf_comment: bool = True,
              ads_compatible: bool = True):
```

because of comments already exists. Sorry for not beeing clear before.